mlflow/pytorch/__init__.py

```diff
@@ -901,6 +902,12 @@ def autolog(
     silent=False,
     registered_model_name=None,
     extra_tags=None,
+    checkpoint=True,
+    checkpoint_monitor="val_loss",
+    checkpoint_mode="min",
+    checkpoint_save_best_only=True,
+    checkpoint_save_weights_only=False,
+    checkpoint_save_freq="epoch",
```

Comment:
what should I do if I want to save checkpoints every 2 epochs?

Reply:
It is not supported yet.

Currently we align the feature with Keras model checkpointing `save_freq` param: https://keras.io/api/callbacks/model_checkpoint/

Every N epoch usage is less common. If it is needed, we can add it in future.

Reply:
How would we support it? Support string like "2 epochs"?

Reply:
yes, I haven't seen anyone doing it. If they really want to, write a custom training loop XD

Reply:
I see, not a big fan of this approach to be honest. I hate arguments that accept multiple types.